--- ./openbabel/Makefile.am.org	Tue Aug 20 16:24:22 2002
+++ ./openbabel/Makefile.am	Sat Sep  7 06:28:34 2002
@@ -2,7 +2,7 @@
 
 METASOURCES=AUTO
 
-KDE_CXXFLAGS = $(USE_RTTI) -DDATADIR=\"$(kde_datadir)/openbabel\"
+KDE_CXXFLAGS = $(USE_RTTI) -DDATADIR=\"$(datadir)/openbabel\"
 INCLUDES = $(all_includes)
 
 lib_LTLIBRARIES = libopenbabel_kemistry.la
@@ -18,6 +18,6 @@
 								   typer.cpp formats.cpp utils.cpp residue.cpp\
 								   cml.cpp
 
-openbabeldatadir = $(kde_datadir)/openbabel
+openbabeldatadir = $(datadir)/openbabel
 openbabeldata_DATA = aromatic.txt atomtyp.txt element.txt extable.txt\
 					 phmodel.txt resdata.txt types.txt
--- ./kembabel/Makefile.am.org	Wed Aug 21 13:50:09 2002
+++ ./kembabel/Makefile.am	Sat Sep  7 06:34:54 2002
@@ -4,11 +4,11 @@
 LDFLAGS = $(all_libraries)
 
 METASOURCES = AUTO
-bin_PROGRAMS = babel kembabel
+bin_PROGRAMS = kembabel
 
-babel_SOURCES = babel.cpp
-babel_LDFLAGS = 
-babel_LDADD = ../openbabel/libopenbabel_kemistry.la
+#babel_SOURCES = babel.cpp
+#babel_LDFLAGS = 
+#babel_LDADD = ../openbabel/libopenbabel_kemistry.la
 
 kembabel_SOURCES = main.cpp kembabel.cpp kembabel-openbabel.cpp \
 				   kembabelwidgetbase.ui
